api/v1alpha1/propagation_types.go

```diff
@@ -140,6 +146,7 @@ type PropagationStatus struct {
 	DeploymentStatus          DeploymentStatus           `json:"deploymentStatus,omitempty"`
 	Conditions                []metav1.Condition         `json:"conditions,omitempty"`
 	DeploymentStatusesReports []DeploymentStatusesReport `json:"deploymentStatusesReports,omitempty"`
+	DeployAfter               DeployAfter                `json:"deployAfter,omitempty"`
```

Comment:
I don't understand how this field is filled now. Is it implemented?

Reply:
it's not implemented yet. it's just an easy way to refactor this. this field should be populated once we can push the config from git repo to OCI registry.